### ISSUE TITLE
Support Harvester version >= 1.1.0

### ISSUE THREAD

**Issue by kazunarikudo** [genicam/harvesters_gui#18]
**Is your feature request related to a problem? Please describe.**
Harvester has introduced backward compatibility at version 1.1.0. Having that incompatibility Harvester GUI versions <= 1.0.n does not work with that.

**Describe the solution you'd like**
Upgrade Harvester GUI so that it works with Harvester version >= 1.1.n.

**Describe alternatives you've considered**
None.

**Additional context**
None.

**Comment by kazunarikudo** [comment]
See also [this ticket](https://github.com/genicam/harvesters/issues/120) about the detail.

**Comment by barriebarry** [comment]
I installed the latest Harvester 1.3 and then I installed Harvester_gui in the same environment.  It then seemed to have removed Harvester 1.3 but then reported successful with installing Harvester 1.0? (I didn't capture the number) along with a bunch of other things.  I'm assuming then this would be in line with the statement above in that the gui is not yet compatible with 1.3.   But now when I tried opening Spyder (5.0) in the same environment, Spyder will not open and reported the following message:

Traceback (most recent call last):
File "C:\Users\robinson\Anaconda3\envs\py37\Scripts\spyder-script.py", line 10, in sys.exit(main())
File "C:\Users\robinson\Anaconda3\envs\py37\lib\site-packages\spyder\app\start.py", line 210, in main from spyder.app import mainwindow
File "C:\Users\robinson\Anaconda3\envs\py37\lib\site-packages\spyder\app\mainwindow.py", line 60, in from qtpy import QtWebEngineWidgets # analysis:ignore
File "C:\Users\robinson\Anaconda3\envs\py37\lib\site-packages\qtpy\QtWebEngineWidgets.py", line 22, in from PyQt5.QtWebEngineWidgets import QWebEnginePage
ValueError: PyCapsule_GetPointer called with incorrect name

So basically, what do I have to do to make the gui work?  Should I create a separate environment, set for Python 3.7, then install harvester_gui first and let it install the required version of harvester and other dependencies?  I would almost expect this will end up killing Spyder as well.  Or can I fix my first py37 environment to make Spyder launch?  I'm doing all this from Anaconda and its power shell, both of which is all new to me.

By the way, what command line do I use to see what version of Harvester is installed?

(Windows 10)
